cves/kernel/CVE-2017-18079.yml

```diff
@@ -55,7 +55,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A vulnerability in the Linux kernel was discovered in that the port->exists
```

Comment:
Either give some more context to explain what port->exists is or replace with a general termanology, like value in a struct. This seems to ba a bit to context specific to include in a description for people who have not looked at the actual code

Reply:
That's a good point. I will change it to general terminology.